doc/source/index.rst

```diff
@@ -51,42 +53,34 @@ It's also possible you end up getting experienced with:
    For a more elaborate introduction to the tools and services that JupyterHub
    depends upon, see our :ref:`tools` page.
 
-
-
 .. _getting-to-zero:
 
-Setup a Kubernetes cluster
---------------------------
+Setup Infrastructure
```

Comment:
For me, this title was a bit vague and I believe adding "cluster" makes it a little bit clear what to expect.
(@consideRatio, what do you think?)
```suggestion
Setup Cluster Infrastructure
```

Reply:
I find "Cluster" to be a overloaded word with too much different interpretations to be helpful, and that someone reading this without already assuming "Kubernetes cluster" when "cluster" is named would benefit from simply hearing "Setup Kubernetes" instead.

Setup Kubernetes / Setup Infrastructure - okay okay with both these options, but lean towards the first to be clearer.

Reply:
Hmmm, I've been thinking about this a bit more, and think we should opt for Setup Kubernetes rather than Setup infrastructure. I'm worried that Setup Infrastructure implies that it will be in scope for this guide to help with for example setting up a dedicated out-of-Helm-chart database to reference in various cloud providers or similarly, but what we actually aim for have have chosen to limit the scope to, is to help setup the required k8s cluster to a functional state, which may of course involve some cloud specific details though.

Reply:
Sure! The only reason I had mentioned Setup Infra was to account for the Helm installation part mentioned. But, this will work out too, with a bit of context on the `Setup JupyterHub` landing page.